Description: Set 'noset' parameter for SessionHandler instance creation during unit tests.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/kronolith-4.2.29/test/Kronolith/TestCase.php
+++ b/kronolith-4.2.29/test/Kronolith/TestCase.php
@@ -47,7 +47,8 @@
             array(
                 '_PARAMS' => array(
                     'user' => 'test@example.com',
-                    'app' => 'kronolith'
+                    'app' => 'kronolith',
+                    'noset' => true,
                 ),
                 'Horde_Alarm' => 'Alarm',
                 'Horde_Cache' => 'Cache',
